core/domain/subscription_services_test.py

```diff
@@ -56,6 +57,20 @@ def setUp(self):
         self.viewer_id = self.get_user_id_from_email(self.VIEWER_EMAIL)
         self.owner_2_id = self.get_user_id_from_email(self.OWNER_2_EMAIL)
 
+    def _get_all_subscribers_of_creators(self, user_id):
```

Comment:
So -- I see that we have a pattern here for doing this sort of thing, but let's not actually do this. I'm not sure that replicating the actual functions here is a good idea; if the actual function changes later then we end up with skew.

Instead, in the test itself, can we get the subscriptions_model and then do stuff like `self.assertEqual(subscriptions_model.subscriber_ids, ...)` or `self.assertIsNone(subscriptions_model.property)`?